### DISCUSSION TITLE
2009/09/27/%E5%9C%A8%E4%B8%8A%E6%B5%B7%E4%B9%A6%E5%9F%8E%E7%9C%8B%E5%88%B0%E6%88%91%E7%9A%84%E4%B9%A6%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#281] [Announcements]
# 2009/09/27/%E5%9C%A8%E4%B8%8A%E6%B5%B7%E4%B9%A6%E5%9F%8E%E7%9C%8B%E5%88%B0%E6%88%91%E7%9A%84%E4%B9%A6%E4%BA%86

今天休假，又没什么事情，于是跑到上海书城去看书。书城把LabVIEW和其它编程语言归在一个区内，LabVIEW在编程区最后面一个柜台跟程序设计理论相关的书挤在一起。

https://blog.qizhen.xyz/2009/09/27/%E5%9C%A8%E4%B8%8A%E6%B5%B7%E4%B9%A6%E5%9F%8E%E7%9C%8B%E5%88%B0%E6%88%91%E7%9A%84%E4%B9%A6%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Sophie
2010年05月9日 20:21 编辑
Congratulations!

回复
Qizhen
2010年05月6日 11:32 编辑
北京大概经常有培训班，不过我也不了解相关信息。可以直接打电话给NI问一下。

回复
liying
2010年04月30日 14:59 编辑
在看你的书, 虽然因为我一点基础也没有, 而且完全自学, 很多地方看着费劲, 但是,却能感觉到里面的东西是真材实料,而不是硬拼凑的.另外, 在北京什么时候有你们Labview 的培训班之类呢? 期盼…

回复
li
2009年09月30日 16:11 编辑
恭喜啦~~~

回复
Kisha
2009年09月28日 18:14 编辑
出书赚了一笔？btw，你也在上海啊…~~

回复
tang
2009年09月28日 17:14 编辑
我也会去看看的，哈哈

回复
gang
2009年09月28日 16:46 编辑
我也买了~~~hiahia

回复
Yi
2009年09月28日 15:48 编辑
先给咱们来个内部签名赠书吧，hiahia

回复
晓柏
2009年09月28日 15:39 编辑
哈哈，不错，大侠要主动出击，给读者一个惊喜：）

回复
Shi
2009年09月28日 15:00 编辑
够满足的 \\(^o^)/~

回复
将军
2009年09月28日 14:02 编辑
哇塞

回复
晖
2009年09月28日 13:46 编辑
那你可以不声张，偷偷的给那几本都签上名好了：）

回复
Qizhen
2009年09月28日 10:49 编辑
没有，我是个低调的人 ^_^*

回复
Coral
2009年09月28日 09:48 编辑
你没跟他说：“要不要我给你签个名啊”？

回复
Hongyan
2009年09月28日 09:38 编辑
赞！我要买来看，O(∩_∩)O

回复
